internal/config/supervisor/config.go

```diff
@@ -18,6 +18,11 @@ import (
 	"go.pinniped.dev/internal/plog"
 )
 
+const (
```

Comment:
Should "tcp" also be a const here? It's the only other allowed setting.

Reply:
Fixed.